### PR TITLE
setup immigration__entitled_to_indefinite_stay

### PR DESCRIPTION
**Pull request by mischa-s**
* Impacted areas: `variables/acts/citizenship/citizenship.py` `variables/acts/immigration/indefinite_stay.py`
* Details:
  - add `immigration__entitled_to_indefinite_stay` for use in citizenship presence calculations 

Ideally I would like to add some more comprehensive testing but basic testing is implemented

### CODE REVIEW

File: openfisca_aotearoa/tests/citizenship/citizenship_adding_days.yaml [hunk 1]
```diff
@@ -12,6 +12,14 @@
         "2015-01-04": true
         "2015-01-05": true
         "2015-01-06": false
+      immigration__entitled_to_indefinite_stay:
```
Comment (Br3nda) [inline openfisca_aotearoa/tests/citizenship/citizenship_adding_days.yaml]: :sparkles:

**Comment by Br3nda** [comment]
This pull request **introduces 3 alerts** when merging 54fe706a29ce1b9981ef4a714e498c5c8c12b895 into b896959b04e424043dc8272e38de6e8d0a16b85e - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/openfisca-aotearoa/rev/pr-42b43cc414c78183bc02c31c3f4b74bb2f4dd5ca)

**new alerts:**

* 2 for Non\-callable called
* 1 for First argument of a method is not named &#39;self&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by mischa-s** [comment]
I've now added a few tests, not sure what LGTM is on about, but the sections it is complaining about are consistent with the style from the rest of the project

**Comment by Br3nda** [comment]
This pull request **introduces 3 alerts** when merging 76ca3220996659c9139866a8d9e690ea67810175 into b896959b04e424043dc8272e38de6e8d0a16b85e - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/openfisca-aotearoa/rev/pr-12c27cbbf9d7fae1f9ceaeaebe378439fd8d2ba0)

**new alerts:**

* 2 for Non\-callable called
* 1 for First argument of a method is not named &#39;self&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by mischa-s** [comment]
@verbman, you happy to merge this?

**Comment by mischa-s** [comment]
approved-cant-merge

**Pull request merged**